routes/web.php

```diff
@@ -17,6 +18,10 @@
 	return config('alttp.randomizer.entrance');
 });
 
+Route::get('base_rom/settings', function () {
+	return ['rom_hash' => Rom::HASH, 'base_file' => elixir('js/base2current.json')];
```

Comment:
generally when doing an array we tend to expand and put each thing on it's own line, e.g.:
```
return [
	'rom_hash' => Rom::HASH,
	'base_file' => elixir('js/base2current.json'),
];
```